src/components/mobileOnlyAccordion/MobileOnlyAccordionButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { useMediaQuery, useMultiStyleConfig } from '@chakra-ui/react';
+
+import Box from '../box';
+import AccordionButton from '../accordion/AccordionButton';
+import { breakpoints } from '../../themes';
+
+const MobileOnlyAccordionButton: FC<PropsWithChildren> = (props) => {
+  const { children, ...rest } = props;
+  const { titleOnDesktop } = useMultiStyleConfig('Accordion');
+  const [isLargerThan1024] = useMediaQuery(
```

Comment:
Whats the benefit compared to using the show/hide component?

Reply:
(Also you are implicitly relying that md will always be 1024 px which may change at some point)

Reply:
After splitting accordion by components Show/Hide stopped working properly, it's not possible to toggle, I assume it's lost context somehow. That's why I changed to `useMediaQuery`. Also I read on chakra issues, that they considering deprecating those components https://github.com/chakra-ui/chakra-ui/issues/6589#issuecomment-1230654365

I just renamed it to `isLargerThanMd` to not specify 1024 and it's relaying on theme breakpoints https://github.com/smg-automotive/components-pkg/pull/355/files#diff-e56f400b12e1e3219bb70d942cbc55a6df4cd84f8f6200e6279e7b4975ebf12aR13
so even if it's changed we are okay